caffe2/core/context_gpu.cu

```diff
@@ -351,6 +351,14 @@ std::mutex& CUDAContext::mutex() {
   return m;
 }
 
+bool CUDAContext::TensorCoreAvailable() {
+  bool available = caffe2::TensorCoreAvailable();
+  if (!available) {
+    LOG(INFO) << "Requesting Tensor Cores when not available";
```

Comment:
I'll make this change when you've decided where you want the single `TensorCoreAvailable()` method to end up.